src/ChainlinkOracle.sol

```diff
@@ -37,12 +37,13 @@ contract ChainlinkOracle is IOracle {
 
     /* CONSTRUCTOR */
 
-    /// @dev Here is the list of assumptions on the inputs that guarantees the oracle behaves as expected:
+    /// @dev Here is the list of assumptions that guarantees the oracle behaves as expected:
     /// - Feeds are either Chainlink-compliant or the address zero.
     /// - Feeds have the same behavioral assumptions as Chainlink's.
     /// - Feeds are set in the correct order.
     /// - Decimals passed as argument are correct.
-    /// - The vault conversion sample is low enough to avoid overflows.
+    /// - The vault's sample assets and the base feed prices don't overflow when multiplied.
```

Comment:
```suggestion
    /// - The converted vault's sample assets and the base feed prices don't overflow when multiplied.
```
because we multiply the conversion of sample assets, not directly the sample assets, with the base feed prices

Reply:
The input is the vault's sample shares so I thought the vault's sample assets was clear enough, but I admit having the keyword concerted makes it clearer
What about "the vault's quoted sample assets"?

Reply:
> The input is the vault's sample shares so I thought the vault's sample assets was clear enough

Oh ok, I missed that. In the same vein "the vault's quoted sample assets" makes it feel like it's the sample assets that are quoted (but it's the sample shares). 

Actually I think we should avoid "sample assets" altogether, because it does not appear in the code, "sample" refers to shares. Maybe "the vault's sample converted as assets" or "the vault's converted sample amount"

Reply:
```suggestion
    /// - The vault's sample shares quoted as assets and the base feed prices don't overflow when multiplied.
```